dlme_airflow/drivers/xml.py

```diff
@@ -16,22 +16,26 @@ class XmlSource(intake.source.base.DataSource):
     def __init__(self, collection_url, dtype=None, metadata=None):
         super(XmlSource, self).__init__(metadata=metadata)
         self.collection_url = collection_url
+        self._record_selector = self._get_record_selector()
         self._path_expressions = self._get_path_expressions()
         self._records = []
 
     def _open_collection(self):
         collection_result = requests.get(self.collection_url).content
         xtree = etree.fromstring(collection_result)
-        for counter, element in enumerate(xtree.findall(".//item"), start=1):
```

Comment:
This was the problem w/ the BNF catalog entry: `.//item` was hard coded and the configured `record_selector` was being ignored.